scripts/prettier-javascript-serializer.js

```diff
@@ -1,6 +1,16 @@
+const fs = require('fs');
+const path = require('path');
 const prettier = require('prettier');
 
-const prettierOptions = {} // optional
+const prettierrc = require('../.prettierrc.js');
+const prettierOptions = Object.assign(
+  {
+    trailingComma:
+      /** @type {import('prettier').ParserOptions['trailingComma']} */ ('es5'),
+  },
+  prettierrc,
+  prettierrc.overrides[0].options
```

Comment:
I'd rather this didn't depend on the global prettierrc to be honest. prettier rc in the root of the project is about how we format our own code. This is about formatting snapshots. There's no particular reason they should be locked together.

Reply:
i've reverted this file to ce24764.

Reply:
slight lie: i had to add `{ parser: 'babel' }` to fix a deprecation warning